service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -216,6 +263,92 @@ void aboutToSubmitShouldPopulateHiddenCCDFieldsInStandardDirectionOrderToPersist
         }
     }
 
+    @BeforeEach
+    void setup() {
+        given(mapperService.mapObject(Mockito.any(), Mockito.any()))
+            .willReturn(CaseData.builder().familyManCaseNumber("12345").respondents1(ImmutableList.of(
+                Element.<Respondent>builder()
+                    .value(Respondent.builder()
+                        .party(RespondentParty.builder()
+                            .lastName("Moley")
+                            .build())
+                        .build())
+                    .build()))
+                .hearingDetails(ImmutableList.of(
+                    Element.<HearingBooking>builder()
+                        .id(UUID.randomUUID())
+                        .value(HearingBooking.builder().date(LocalDate.of(2020,10, 27)).build())
+                        .build())).build());
+
+        given(localAuthorityNameLookupConfiguration.getLocalAuthorityName(Mockito.any()))
```

Comment:
why do we need to use any for getLocalAuthorityName? We have an integration-test profile for variables where LA name should be Example Local Authority anyway